core/model/src/main/kotlin/org/meshtastic/core/model/util/DateTimeUtils.kt

```diff
@@ -62,5 +64,26 @@ private fun formatUptime(seconds: Long): String {
         .joinToString(" ")
 }
 
-@Suppress("MagicNumber")
-fun onlineTimeThreshold() = (System.currentTimeMillis() / 1000 - 2 * 60 * 60).toInt()
+fun onlineTimeThreshold(): Int {
+    val currentSeconds = System.currentTimeMillis() / TimeUnit.SECONDS.toMillis(1)
+    return (currentSeconds - ONLINE_WINDOW_HOURS * TimeUnit.HOURS.toSeconds(ONLINE_WINDOW_HOURS.toLong())).toInt()
+}
+
+/**
+ * Formats remaining mute time in a human-readable format
+ *
+ * @param remainingMillis The remaining time in milliseconds
+ * @return Formatted string as "2d, 3.5h" or "Not muted" if time has expired
+ */
+fun formatMuteRemainingTime(remainingMillis: Long): String {
+    if (remainingMillis <= 0) return "Not muted"
```

Comment:
Use the string resource

Reply:
It should never get that - it get's intercepted at the level above, and this file doesn't have access to strings from memory. 

i could assert, but that seems less pleasent.